### PR TITLE
add zeit warning

### PR DESCRIPTION
**Pull request by SaraVieira**
**What kind of change does this PR introduce?**
Feature 

**What is the current behavior?**
No warning saying we only support version 1 of zeit

**What is the new behavior?**
Adds a new warning for version 1 and also a way for people to say they want version 2

zeit2

It also shows a notification saying your feedback was saved but that's not on the gif

### CODE REVIEW

**Comment by CompuIves** [comment]
Omg I *love* this!

File: packages/app/src/app/store/modules/deployment/actions.js [hunk 1]
```diff
@@ -14,6 +15,51 @@ export function loadZip({ props, jsZip }) {
   return jsZip.loadAsync(file).then(result => ({ contents: result }));
 }
 
+export function getPeopleWhoWant2() {
```
Comment (CompuIves) [inline packages/app/src/app/store/modules/deployment/actions.js]: Could we perhaps make this function async and dynamically import `airtable`? That should shave us some bundle size 😄

Comment (SaraVieira) [reply packages/app/src/app/store/modules/deployment/actions.js]: You want a fancy require inside that ? 😄

File: packages/app/src/app/store/modules/deployment/actions.js [hunk 2]
```diff
@@ -1,5 +1,5 @@
 import { omit } from 'lodash-es';
-
+import Loadable from 'app/utils/Loadable';
```
Comment (CompuIves) [inline packages/app/src/app/store/modules/deployment/actions.js]: I think that this one can be removed.

Comment (SaraVieira) [reply packages/app/src/app/store/modules/deployment/actions.js]: 🤦‍♂️

**Pull request merged**